pyspark/bigdl/keras1/converter.py

```diff
@@ -403,16 +408,218 @@ def create_reshape(self, klayer, kclayer):
+        paddinglayer1 = BLayer.Padding(dim=1,
+                                       pad=-padding[0],
+                                       n_input_dim=len(input_shape) - 1,
+                                       value=0.0,
+                                       n_index=1,
+                                       bigdl_type="float")
+        paddinglayer2 = BLayer.Padding(dim=1,
+                                       pad=padding[0],
+                                       n_input_dim=len(input_shape) - 1,
+                                       value=0.0,
+                                       n_index=1,
+                                       bigdl_type="float")
+        paddinglayer3 = BLayer.Padding(dim=2,
+                                       pad=-padding[1],
+                                       n_input_dim=len(input_shape) - 1,
+                                       value=0.0,
+                                       n_index=1,
+                                       bigdl_type="float")
+        paddinglayer4 = BLayer.Padding(dim=2,
+                                       pad=padding[1],
+                                       n_input_dim=len(input_shape) - 1,
+                                       value=0.0,
+                                       n_index=1,
+                                       bigdl_type="float")
+        paddinglayer5 = BLayer.Padding(dim=3,
+                                       pad=-padding[2],
+                                       n_input_dim=len(input_shape) - 1,
+                                       value=0.0,
+                                       n_index=1,
+                                       bigdl_type="float")
+        paddinglayer6 = BLayer.Padding(dim=3,
+                                       pad=padding[2],
+                                       n_input_dim=len(input_shape) - 1,
+                                       value=0.0,
+                                       n_index=1,
+                                       bigdl_type="float")
+        model.add(paddinglayer1)
+        model.add(paddinglayer2)
+        model.add(paddinglayer3)
+        model.add(paddinglayer4)
+        model.add(paddinglayer5)
+        model.add(paddinglayer6)
+        return model
+
+    # Cropping layers
+    # Only cropping1d is supported for now.
+    def create_cropping1d(self, klayer, kclayer):
+        cropping = tuple(klayer.cropping)
+        return BLayer.SpatialZeroPadding(0, 0, -cropping[0], -cropping[1])
+
+    # Recurrent Layers
+    def create_simplernn(self, klayer, kclayer):
+        rec = BLayer.Recurrent()
+        input_shape = klayer.get_input_shape_at(0)
+        config = kclayer["config"]
+        activation = self.to_bigdl_activation(config["activation"],
+                                              "%s_%s" % (config["name"], config["activation"]))
+        print(input_shape[2])
+        print(klayer.output_dim)
+        rnn = BLayer.RnnCell(input_size=input_shape[2],
```

Comment:
how to handle `init='glorot_uniform', inner_init='orthogonal',` and `dropout_W=0., dropout_U=0.` ?